app/ui-master/public/icons/omnichannel.svg

```diff
@@ -1,3 +1,3 @@
-<svg id="livechat" viewBox="0 0 20 20">
+<svg id="omnichannel" viewBox="0 0 20 20">
```

Comment:
is this change on purpose? doesn't seem related

Reply:
Yes, @renatobecker edited the `icons.svg` to change the icon's id from livechat to omnichannel in this [commit](https://github.com/RocketChat/Rocket.Chat/commit/17b1535a16c18b95b0836a1e271bb40940dbe7fd#diff-501ea7c311e0971aaa6decbe11d7df79R200) directly. We added a new icon called `game.svg` and run update icons scripts in this commit. To avoid overwriting his changes, so I changed the name and id of the livechat icon and run the update icons scripts again.

Reply:
ok then =) thanks